src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -41,20 +41,43 @@ const TutorProfile = () => {
     defaultResponse: defaultResponses.array
   })
 
+  useEffect(() => {
+    if (
+      !loading &&
+      response &&
+      response.videoLink &&
+      response.videoLink.tutor !== null &&
+      response.videoLink.tutor !== ''
+    ) {
```

Comment:
This condition looks a bit confusing. Can we refactor to this ?
`!loading && response?.videoLink?.tutor``

Reply:
I agree. 
we can replace it with this expression 
!loading &&
 response?.videoLink?.tutor !== null &&
 response?.videoLink?.tutor !== ' '
but we can't stop checking for null and empty string.

Reply:
You can also split this expression into separate variables to improve code readability

Reply:
fixed 
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/63c9c8f2-2b23-43cf-b986-013cdd4a50d8)

Reply:
what a valid falsy value are you expecting here, that just `... && tutorVideo` is not enough?